src/soilbiogeochem/SoilBiogeochemDecompCascadeBGCMod.F90

```diff
@@ -405,54 +405,54 @@ subroutine init_decompcascade_bgc(bounds, soilbiogeochem_state_inst, soilstate_i
               errMsg(sourcefile, __LINE__))
       end if
 
-      i_pro_som = i_lig_lit + 1
-      floating_cn_ratio_decomp_pools(i_pro_som) = .false.
-      decomp_cascade_con%decomp_pool_name_restart(i_pro_som) = 'soil1'
-      decomp_cascade_con%decomp_pool_name_history(i_pro_som) = 'SOIL1'
-      decomp_cascade_con%decomp_pool_name_long(i_pro_som) = 'soil 1'
-      decomp_cascade_con%decomp_pool_name_short(i_pro_som) = 'S1'
-      is_litter(i_pro_som) = .false.
-      is_soil(i_pro_som) = .true.
-      is_cwd(i_pro_som) = .false.
-      initial_cn_ratio(i_pro_som) = cn_s1
-      initial_stock(i_pro_som) = params_inst%initial_Cstocks(i_pro_som)
-      is_metabolic(i_pro_som) = .false.
-      is_cellulose(i_pro_som) = .false.
-      is_lignin(i_pro_som) = .false.
-
-      i_rec_som = i_pro_som + 1
-      floating_cn_ratio_decomp_pools(i_rec_som) = .false.
-      decomp_cascade_con%decomp_pool_name_restart(i_rec_som) = 'soil2'
-      decomp_cascade_con%decomp_pool_name_history(i_rec_som) = 'SOIL2'
-      decomp_cascade_con%decomp_pool_name_long(i_rec_som) = 'soil 2'
-      decomp_cascade_con%decomp_pool_name_short(i_rec_som) = 'S2'
-      is_litter(i_rec_som) = .false.
-      is_soil(i_rec_som) = .true.
-      is_cwd(i_rec_som) = .false.
-      initial_cn_ratio(i_rec_som) = cn_s2
-      initial_stock(i_rec_som) = params_inst%initial_Cstocks(i_rec_som)
-      is_metabolic(i_rec_som) = .false.
-      is_cellulose(i_rec_som) = .false.
-      is_lignin(i_rec_som) = .false.
-
-      i_avl_som = i_rec_som + 1
-      floating_cn_ratio_decomp_pools(i_avl_som) = .false.
-      decomp_cascade_con%decomp_pool_name_restart(i_avl_som) = 'soil3'
-      decomp_cascade_con%decomp_pool_name_history(i_avl_som) = 'SOIL3'
-      decomp_cascade_con%decomp_pool_name_long(i_avl_som) = 'soil 3'
-      decomp_cascade_con%decomp_pool_name_short(i_avl_som) = 'S3'
-      is_litter(i_avl_som) = .false.
-      is_soil(i_avl_som) = .true.
-      is_cwd(i_avl_som) = .false.
-      initial_cn_ratio(i_avl_som) = cn_s3
-      initial_stock(i_avl_som) = params_inst%initial_Cstocks(i_avl_som)
-      is_metabolic(i_avl_som) = .false.
-      is_cellulose(i_avl_som) = .false.
-      is_lignin(i_avl_som) = .false.
+      i_pas_som = i_lig_lit + 1
```

Comment:
This should be active SOM (S1)?  Passive should be the last pool (S3).

Reply:
Thank you for catching @wwieder! I will correct this later today.